test_scripts/API/VehicleData/OnVehicleData/002_NotificationForUnsubsribedParameter_Ignored_flow.lua

```diff
@@ -0,0 +1,95 @@
+---------------------------------------------------------------------------------------------------
+-- Item: Use Case: request is allowed by Policies
+--
+-- Requirement summary:
+-- [SubscribeVehicleData] As a mobile app wants to send a request to subscribe for specified parameter
+--
+-- Description:
+-- In case:
+-- 1) hmi application sends valid SubscribeVehicleData to SDL and this request is allowed by Policies
+-- SDL must:
+-- Transfer this request to HMI and after successful response from hmi
+-- Respond SUCCESS, success:true to mobile application
+-- After HMI sends notification about changes in subcribed parameter
+-- SDL must:
+-- Forward this notification to mobile application
```

Comment:
@AByzhynar Description needs to be updated according to the script

Reply:
@dboltovskyi Fixed in 3b703abdab25b3e13b150954e8a8c707f40a3b45